pkg/repositories/gormimpl/join.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	joinCondition = "JOIN %s ON %s"
+	joinCondition = "JOIN %s %s ON %s"
```

Comment:
at the risk of being super annoying would you mind adding a comment on what the expected or example values are here since it's not immediately obvious?

Reply:
not annoyed at all, done